### PR TITLE
[FLINK-23490][examples][table] - Updated print output

### PR DESCRIPTION
**Pull request by mans2singh**
## What is the purpose of the change

* StreamWindowSQLExample print output shows older format

## Brief change log

* Updated print output in the example

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java [hunk 1]
```diff
@@ -87,8 +87,8 @@ public static void main(String[] args) throws Exception {
         env.execute("Streaming Window SQL Job");
 
         // should output:
-        // 2019-12-12 00:00:00.000,3,10,3
-        // 2019-12-12 00:00:05.000,3,6,2
+        // +I[2019-12-12 00:00:05.000, 3, 6, 2]
+        // +I[2019-12-12 00:00:00.000, 3, 10, 3]
```
Comment (tsreaper) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java:91]: It seems that the output order is reversed. Is there any way to enforce the output order?

**Review by tsreaper — COMMENTED** [review]
Thanks for contributing to Flink! I've left a comment.

**Review by JingsongLi — COMMENTED** [review]
Thanks!
Maybe we can just:
```
tEnv.executeSql(query).print();
```
Then will be:
```
+----+--------------------------------+----------------------+--------------+----------------------+
| op |                   window_start |            order_num | total_amount |      unique_products |
+----+--------------------------------+----------------------+--------------+----------------------+
| +I |        2019-12-12 00:00:00.000 |                    3 |           10 |                    3 |
| +I |        2019-12-12 00:00:05.000 |                    3 |            6 |                    2 |
+----+--------------------------------+----------------------+--------------+----------------------+
```

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java (hunk 1 above)
Comment (mans2singh) [reply flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java:91]: @tsreaper - I've tried it several times and the order of the output is not maintained.  Please let me know if you have any recommendation.  Thanks.

Comment (mans2singh) [reply flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java]: @tsreaper - I've changed the parallelism to 1 and not the output is in the same order as you had mentioned.  I've also added table output recommended by @JingsongLi.
Please let me know if you have any other recommendations.  Thanks

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java [hunk 2]
```diff
@@ -81,14 +82,23 @@ public static void main(String[] args) throws Exception {
                         + "GROUP BY TUMBLE(ts, INTERVAL '5' SECOND)";
         Table result = tEnv.sqlQuery(query);
```
Comment (JingsongLi) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java:83]: We can remove old codes.

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java (hunk 1 above)
Comment (tsreaper) [reply flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java]: I suppose using one of them is enough. The one used by the original example is using the PrintSink which may have multiple instances in a job and may produce out of order results (unless the user explicitly sets the parallelism to 1), the other one mentioned by Jingsong is using the CollectSinkFunction and it's parallelism is always 1 (no matter what the parallelism is in other parts of that job), thus there is no out of order issue.

**Comment by mans2singh** [comment]
@tsreaper @JingsongLi - I've removed the old print code as you recommended.  Please let me know if you have additional comments.  Thanks for your time.

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java [hunk 3]
```diff
@@ -40,6 +38,7 @@
     public static void main(String[] args) throws Exception {
         // set up execution environment
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.setParallelism(1);
```
Comment (JingsongLi) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java:41]: We don't need this line too. Table print will keep the order.

**Comment by mans2singh** [comment]
@flinkbot run azure

**Review by tsreaper — APPROVED** [review]
Looks good to me. @JingsongLi any other thoughts?

File: flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java [hunk 4]
```diff
@@ -79,16 +77,16 @@ public static void main(String[] args) throws Exception {
...
                         + "FROM orders\n"
                         + "GROUP BY TUMBLE(ts, INTERVAL '5' SECOND)";
-        Table result = tEnv.sqlQuery(query);
-        tEnv.toAppendStream(result, Row.class).print();
 
-        // after the table program is converted to DataStream program,
-        // we must use `env.execute()` to submit the job.
-        env.execute("Streaming Window SQL Job");
+        tEnv.executeSql(query).print();
+        // should output;
```
Comment (JingsongLi) [inline flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java:82]: `should output:`

Comment (mans2singh) [reply flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java]: @JingsongLi - I've corrected the code based on your advice.  Please let me know you have any additional comments.  Thanks again for your review/comments.

**Review by JingsongLi — APPROVED** [review]
Thanks for the update. Looks good to me!

**Comment by mans2singh** [comment]
Thanks @JingsongLi @tsreaper for your help.

**Pull request merged**